view/field/index.ts

```diff
@@ -89,13 +89,35 @@ function removeByPosition(str: string, position: number): string {
   return str.slice(0, position) + str.slice(position + 1)
 }
 
+const highlightControl = {
```

Comment:
Let’s split to 2 simple functions `highlightControl` and `unhighlightControl`, it will be better for JS compression